packages/block-editor/src/components/preview-options/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	Dropdown,
+	MenuGroup,
+	MenuItem,
+	Path,
+	SVG,
+} from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { check } from '@wordpress/icons';
+
+const downArrow = (
```

Comment:
Is this same as https://github.com/WordPress/gutenberg/blob/master/packages/icons/src/library/arrow-down.js ?

Reply:
No, that one looks different, you can check it out [here](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library) (also the `path` value was different in that code compared to what we have here).

Reply:
So this  is  `chevronDown`  right?

Reply:
> So this is chevronDown right?

Oh right, missed that one. I could pass it to icon prop of the button component, but that would place it on the left side of the text, instead of right where it's currently place. What do you think? Btw. for some reason when I replace the existing `downArrow` with `chvronDown` here nothing gets rendered. 🤔